[     3.395][MainProcess][INFO] Parsing specification...
[     3.401][MainProcess][DEBUG] Reading input files...
[     3.411][MainProcess][INFO] Inferred data types for table tests-examples/scythe/recent_posts/tables/024.csv: ['int64', 'string', 'string']
[     3.417][MainProcess][INFO] Inferred data types for table tests-examples/scythe/recent_posts/tables/024_o.csv: ['string', 'float64']
[     3.448][MainProcess][DEBUG] Generating DSL...
[     3.464][MainProcess][INFO] Creating 2 processes
[     3.474][MainProcess][DEBUG] Increasing generator loc to 2
[     3.474][MainProcess][DEBUG] Increasing generator loc to 3
[     3.476][MainProcess][DEBUG] Increasing generator loc to 4
[     3.497][MainProcess][DEBUG] Increasing generator loc to 5
[     3.692][MainProcess][DEBUG] Increasing generator loc to 6
[     5.677][MainProcess][DEBUG] Increasing generator loc to 7
[    26.051][MainProcess][DEBUG] Increasing generator loc to 8
[   235.623][cube-solver-0][DEBUG] Initialising process for 8 lines of code.
Traceback (most recent call last):
  File "./cubes.py", line 95, in <module>
    main()
  File "./cubes.py", line 87, in main
    program = synthesizer.synthesize()
  File "/home/rmbb/CUBES/squares/parallel_synthesizer.py", line 250, in synthesize
    cube = next(generator)
  File "/home/rmbb/CUBES/squares/dsl/dc.py", line 131, in __next__
    while not self.line_stack[-1]:
IndexError: list index out of range
